package/kedro_viz/api/rest/responses/utils.py

```diff
@@ -16,27 +16,13 @@
 
 
 # Shared status enums for pipeline, nodes, and datasets
-class PipelineStatus(str, Enum):
+class Status(str, Enum):
```

Comment:
```suggestion
class RunEventStatus(str, Enum):
```

Status seems very generic. May be something related to the events should provide clear information.